download/releaseNotes/releaseNotes7.2.adoc

```diff
@@ -0,0 +1,69 @@
+= Release Notes 7.2
+:awestruct-description: New and noteworthy, demos and status for OptaPlanner 7.2.
+:awestruct-layout: releaseNotesBase
+:awestruct-priority: 1.0
+:awestruct-release_notes_version: 7.2
+:awestruct-release_notes_version_qualifier: Final
+
+[[NewAndNoteWorthyEngine]]
+== New and noteworthy: Engine
+
+=== Construction Heuristic power tweaking made simpler
+
+It's now easier to power tweak the Construction Heuristic's move selectors if desired.
+This especially useful to improve scaling for use cases with multiple planning variables.
+
+For example, in course scheduling this will first assign a period to lecture and then a room to a lecture
+(instead of the combination of both which is the default behaviour):
+
+[source,xml,options="nowrap"]
+----
+  <constructionHeuristic>
+    <constructionHeuristicType>FIRST_FIT_DECREASING</constructionHeuristicType>
+    <changeMoveSelector>
+      <valueSelector variableName="period"/>
+    </changeMoveSelector>
+    <changeMoveSelector>
+      <valueSelector variableName="room"/>
+    </changeMoveSelector>
+  </constructionHeuristic>
+----
+
+This means that it will evaluate a lot less moves, resulting in a Construction Heuristic that end much quicker
+at the cost of solution quality (which Local Search can usually make up for in the time gained).
+
+[[NewAndNoteWorthyWorkbench]]
+== New and noteworthy: Workbench
+
+=== Guided decision tables integration
+OptaPlanner now integrates with the Guided decision table editor.
+Modify the score with a build-in OptaPlanner actions that can be accessed in the `Action BRL fragment` column type.
+
+image:7.2/guidedDecisionTable.png[Guided decision table]
+
+
+[[NewAndNoteWorthyExecutionServer]]
+== New and noteworthy: Execution Server
+
+=== Real-time planning
+The KIE Server now exposes an interace to update a problem dataset while the solver is running.
+Use Java client or REST interface to submit your ProblemFactChange implementations.
+
+==== Java Client
+[source,java,options="nowrap"]
+----
+  void addProblemFactChange(String containerId,
+                            String solverId,
+                            ProblemFactChange problemFactChange);
+
+  void addProblemFactChanges(String containerId,
+                             String solverId,
+                             List<ProblemFactChange> problemFactChange);
+
+  Boolean isEveryProblemFactChangeProcessed(String containerId,
+                                            String solverId);
+----
+
+==== REST API
+- POST /containers/containerId/solvers/solverId/problemfactchanges
```

Comment:
Can we add something like "new REST API" or "new methods" or "these endpoints have been added"? Now it's just a list, so it's not clear what the change is from 7.1 to 7.2.

Also, let's put the uri's between `backticks`?